qcodes_contrib_drivers/drivers/Standa/Standa_10MWA168.py

```diff
@@ -0,0 +1,211 @@
+from qcodes import Instrument
+import ctypes
+import numpy as np
+import os
+import time
+
+
+class DeviceEnumeration(ctypes.BigEndianStructure):
+    pass
```

Comment:
The docs build fails because there are no docs for the `BigEndianStructure` in the main python docs. 

Do you know the purpose of `DeviceEnumeration`? It does not seem to be used anywhere in the code. 
In that case it may be easier to just delete it. 

Otherwise you can add an ignore to `nitpick_ignore` in  `docs/conf.py` 
I think it should be `('py:class', 'ctypes._endian.BigEndianStructure')`

Reply:
Oh yes, it really isn't used anywhere. Since I got the drivers from someone else, I do not know what purpose this class should serve. So I'll simply remove it.